build/pkgs/cython/version_requirements.txt

```diff
@@ -0,0 +1 @@
+cython
```

Comment:
The reintroduction of these version requirement files containing no actual requirements (which however are present in the pyproject.toml file) seems pretty pointless. This also reverts the design of https://github.com/sagemath/sage/pull/36982 - which already went through the "disputed" process.

Just add all packages whose version is now specified in the pyproject toml in the list in `bootstrap`.

Reply:
> seems pretty pointless.

No. This defines the map between SPKG names and Python distribution names.

Reply:
> Just add all packages whose version is now specified in the pyproject toml in the list in `bootstrap`.

No.

Reply:
> ... This also reverts the design of #36982 - which already went through the "disputed" process.

As I understand it, by the design of #36982:

`src/pyproject.toml` specifies version constraints of the dependencies of the distribution package `sagemath-standard`, independently from the sage-distribution.

This PR respects that, and also reintroduces `build/pkgs/*version_requirements.txt` files to specify further version constraints of the sage packages for the sage-distribution 

@mkoeppe Am I right?

Reply:
@kwankyu That's correct.